### PR TITLE
Update item-full.js

### PR DESCRIPTION
**Pull request by h0dlr**
## Checklist

- [ ] Are your changes backwards compatible?

- [ ] Did you QA this? Could we deploy this straight to production?

- [ ] For frontend changes: Tested on mobile?

## Summary by CodeRabbit

- **Refactor**
	- Improved the structure of content display in the `ItemComponent`.

### CODE REVIEW

**Comment by h0dlr** [comment]
ref https://github.com/stackernews/stacker.news/issues/884#issuecomment-2028894131

**Comment by felipebueno** [comment]
@h0dlr the tags are not being close at the correct place. Unfortunately I'm not able to reproduce the bug so I can't fix it 

image

**Comment by felipebueno** [comment]
Also, I think you created your patch branch from an old version of the master branch. I tried to run it but it didn't work

image

File: components/item-full.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ function TopLevelItem ({ item, noReply, ...props }) {
   const { ref: textRef, quote, quoteReply, cancelQuote } = useQuoteReply({ text: item.text })
 
   return (
+   <article className={styles.fullItemContainer} ref={textRef}>
```
Comment (felipebueno) [inline components/item-full.js:129]: the tags are not being closed at the correct place

**Review by felipebueno — CHANGES_REQUESTED** [review]
The tags are not being closed at the correct place. Unfortunately I'm not able to reproduce the bug so I can't fix it.

Also, I think you created your patch branch from an old version of the master branch. I tried to run it but it didn't work

**Comment by h0dlr** [comment]
Yep that's what I though, sorry I'm unable to test myself. Just wanted to highlight that `<h1>` could be included inside the `<article>` and the item text wrapped in a `<p>`as described in the [reference](https://github.com/stackernews/stacker.news/issues/884#issuecomment-2028894131). The output could look like in the image attached below

output

**Comment by huumn** [comment]
Hey @h0dlr thanks for giving this a shot. 

If you are unable to run the code and test it, perhaps you can describe the problem you're trying to solve in greater detail so someone who can test it can give it a shot?

**Comment by h0dlr** [comment]
Hey @huumn, just submitted a new one with the correct syntax https://github.com/stackernews/stacker.news/pull/1021 including `ItemComponent` inside `<article>`

**Comment by huumn** [comment]
Did you test it and lint it?

**Pull request closed without merging**